sdk/anomalydetector/ai-anomaly-detector/package.json

```diff
@@ -83,6 +83,7 @@
     "@types/node": "^8.0.0",
     "chai": "^4.2.0",
     "cross-env": "^7.0.2",
+    "csv-parse": "^4.4.0",
```

Comment:
should this be moved to the samples's own `package.json`?

Reply:
It should be added there _in addition_ to here. I assumed it was already there, but I will make a change to fix it.

Reply:
so every dep needed by a sample should also be added to the package's devDependencies? hmm

Reply:
I think csv-parse is already in the samples package,json. What's the reason for it having to be in the main package.json? Is it so that rush is able work with the samples? I think it should be fine since it is under devDependencies. I just want to learn the reason.

Reply:
@deyaaeldeen Yes, it is so that sample dependencies will be available when the samples are compiled/run. We don't install the samples package.json, and doing so in a clean way is difficult (we would have to add _every_ samples package.json to Rush, or do something even worse and hacked together).

Conceptually, the sample dependencies are required to build and run the sample code in the development workflow, therefore they are dev dependencies of the package.